angular/src/app/components/login-page/login-page.component.ts

```diff
@@ -26,20 +26,32 @@ export class LoginPageComponent implements OnDestroy{
     this.valueChanged.emit({param:0});
   }
 
+  playAudio() {
+    const audio = new Audio('../assets/loginAudio.mp3');
+    audio.autoplay = true;
+    audio.muted = false;
+    audio.loop = true;
+
+    audio.play().catch(error => {
+      console.error('Error playing audio:', error);
+    });
+  }
+
   selectTab(tab: string) {
     this.selectedTab = tab;
   }
 
   onJoinSession() {
-    const createPlayerDto = { username: this.username, sessionCode: this.sessionCode };
+    const createPlayerDto = { username: this.username, code: this.sessionCode };
     this.sessionService.joinLobby(createPlayerDto);
     setTimeout(()=>{
-      if(this.sessionService.getPlayerUsername() && this.sessionService.getSessionCode())
+      if(this.sessionService.getPlayerUsername()!=undefined && this.sessionService.getSessionCode())
```

Comment:
nema razloga da dodajes != undefined, jer ukoliko je undefined onda i this.sessionService.getPlayerUsername() nece biti tacno u jsu.

Reply:
reseno, razlog je dodeljivanje vrednosti za username player?.username, umesto player!.username u /app/session.service.ts